docs/zkapps/o1js-reference/README.md

```diff
@@ -32,6 +32,6 @@ See the [Contributing guidelines](https://github.com/o1-labs/o1js/blob/main/CONT
 
 High-quality community packages from open source developers are available for your project.
 
-- **snarkyjs-elgamal** A partially homomorphic encryption library for o1js based on Elgamal encryption: [GitHub](https://github.com/Trivo25/snarkyjs-elgamal) and [npm](https://www.npmjs.com/package/snarkyjs-elgamal)
```

Comment:
per https://github.com/o1-labs/docs2/blob/main/CONTRIBUTING.md#o1js-reference this page is auto-generated from code https://github.com/o1-labs/o1js/blob/main/README.md and updates with the next o1js release (can you remove this file from the PR please?)

Reply:
done